libminifi/include/utils/GeneralUtils.h

```diff
@@ -35,7 +35,7 @@ namespace utils {
 #if __cplusplus < 201402L
 template<typename T, typename... Args>
 std::unique_ptr<T> make_unique(Args&&... args) {
-  return std::unique_ptr<T>{ new T{ std::forward<Args>(args)... } };
+  return std::unique_ptr<T>{ new T(std::forward<Args>(args)...) };
```

Comment:
most significant differences between direct-initialization and direct-list-initialization:
- direct-list-initialization doesn't allow narrowing conversions
- direct-list-initialization prefers the initializer-list constructor, and only attempts regular ctor calls when the initializer-list one does not match, as opposed to doing regular overload resolution with the initializer-list constructor in the overload set.